src/components/Rating/Rating.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import styles from './Rating.scss';
+import star from './img/star.svg';
+
+interface RatingProps {
+  rating: number;
+}
+
+const Rating: React.FC<RatingProps> = ({ rating }: RatingProps) => {
```

Comment:
it is not needed basically to set props interface like ` ({ rating }: RatingProps)`

you already did it in  `React.FC<RatingProps>`